### DISCUSSION TITLE
The world database is too difficult to import

### DISCUSSION THREAD

**Discussion by hua445633972** [Legends-of-Azeroth/Legends-of-Azeroth-Pandaria-5.4.8#147] [General]
world_548_20230527 world database, It’s too difficult. I changed to mysql5.7 and 8.0 and MariaDB10.11, but I couldn’t import them. How could it be so difficult?

**Comment by AventusDev** [comment]
Hi, you can try SQLDumpSplitter3 for splitting database in few sql chunks and then import it one by one.

**Reply by fuzionproject** [reply]
Modify on config mysql to set the limit read and write more up as 126 M to work

**Reply by leelf00** [reply]
I test import using mysql client command line tool and navicat.Try disable transaction when you import it.
